docs/packages/pkg/debug_assert.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    debug_assert
+
+.. index:: unsorted ; debug_assert
+
+.. _pkg.debug_assert:
+
+debug_assert
+===
```

Comment:
Should be
```rst
debug_assert
============
```

Reply:
I guess you should fix foo.rst then cause I copied from there